debian/changelog

```diff
@@ -1,23 +1,26 @@
-cloud-init (23.1-0ubuntu1~22.10.1) UNRELEASED; urgency=medium
+cloud-init (23.1.1-0ubuntu1~22.10.1) UNRELEASED; urgency=medium
```

Comment:
Although it doesn't really matter as we will replace this UNRELEASED version before actual SRU upload anyway with somethng like 23.1.2-0ubuntu-~22.10.1), we probably should at least use a version that is at least > the previous version in this changelog `23.1.1-0ubuntu1~22.10.1`

```suggestion
cloud-init (23.1.1-0ubuntu1~22.10.2) UNRELEASED; urgency=medium
```